python/ctsm/site_and_regional/plumber_site.py

```diff
@@ -0,0 +1,124 @@
+class Plumber2Site(TowerSite):
+    """
+    A class for encapsulating plumber sites.
+    """
+
+    def build_base_case(
+        self,
+        cesmroot,
+        output_root,
+        res,
+        compset,
+        user_mods_dirs=None,
+        overwrite=False,
+        setup_only=False,
+    ):
+        if user_mods_dirs is None:
+            user_mods_dirs = [
+                os.path.join(self.cesmroot, "cime_config", "usermods_dirs", "PLUMBER2", self.name)
+            ]
+        case_path = super().build_base_case(cesmroot, output_root, res, compset, user_mods_dirs)
+
+        return case_path
+
+    # pylint: disable=too-many-statements
+    def run_case(
+        self,
+        base_case_root,
+        run_type,
+        prism,
+        user_version,
+        tower_type=None,
+        user_mods_dirs=None,
+        overwrite=False,
+        setup_only=False,
+        no_batch=False,
+        rerun=False,
+        experiment=False,
+    ):
+        """
+        Run case.
+
+        Args:
+        self
+        base_case_root: str, opt
+            file path of base case
+        run_type: str, opt
+            transient, post_ad, or ad case, default ad
+        prism: bool, opt (ad case is default because PLUMBER requires spinup)
+            if True, use PRISM precipitation, default False
+        user_version: str, opt
+            default 'latest'; this could be useful later
+        overwrite: bool, opt
+            default False
+        setup_only: bool, opt
+            default False; if True, set up but do not run case
+        no_batch: bool, opt
+            default False
+        rerun: bool, opt
+            default False
+        experiment: str, opt
+            name of experiment, default False
+        """
+        user_mods_dirs = [
+            os.path.join(self.cesmroot, "cime_config", "usermods_dirs", "PLUMBER2", self.name)
+        ]
+        tower_type = "PLUMBER"
+        super().run_case(
+            base_case_root,
+            run_type,
+            prism,
+            user_version,
+            tower_type,
+            user_mods_dirs,
+            overwrite,
+            setup_only,
+            no_batch,
+            rerun,
+            experiment,
+        )
+
+    def set_ref_case(self, case):
+        super().set_ref_case(case)
+        return True  ### Check if super returns false, if this will still return True?
+
+    def modify_user_nl(self, case_root, run_type, rundir, site_lines=None):
+        """
+        This function can be used to include any plumber-specific user namelist lines
+        """
+        # This is just an example currently. TODO: remove?
+        if site_lines is None:
+            site_lines = [
+                """hist_fincl1 = 'TOTECOSYSC', 'TOTECOSYSN', 'TOTSOMC', 'TOTSOMN', 'TOTVEGC',
+                                 'TOTVEGN', 'TLAI', 'GPP', 'CPOOL', 'NPP', 'TWS', 'H2OSNO',"""
+            ]
+        super().modify_user_nl(case_root, run_type, rundir, site_lines)
```

Comment:
I don't think this is needed for PLUMBER, and it also be not be used in NEON.  Could it be removed from one or both *_site.py scripts?

Reply:
This has been removed.